Makefile-ponyc

```diff
@@ -291,7 +291,20 @@ ifneq ($(MAKECMDGOALS),clean)
   # (2) the linker flags necessary to link against the prebuilt libraries
   # (3) a list of include directories for a set of libraries
   # (4) a list of the libraries to link against
-  llvm.ldflags := -L$(CROSS_SYSROOT)$(subst -L,,$(shell $(LLVM_CONFIG) --ldflags $(LLVM_LINK_STATIC)))
+  llvm.libdir := $(CROSS_SYSROOT)$(subst -L,,$(shell $(LLVM_CONFIG) --ldflags $(LLVM_LINK_STATIC)))
+  llvm.ldflags := -L$(llvm.libdir)
+
+  # Set rpath for ponyc if we're dynamically linking LLVM_VENDOR
+  ifeq (,$(filter $(use), llvm_link_static))
+    # We're linking dynamically as "llvm_link_static" is empty.
+    ifeq ($(LLVM_VENDOR),true)
+      # LLVM_VENDOR is true, so have linker set rpath.
+      llvm.ldflags += -Wl,-rpath,$(llvm.libdir)
+    endif
+  endif
+
+  $(warning llvm.libdir="$(llvm.libdir)")
```

Comment:
I'm missing some context, and warning might be the only "log level" available. If there's a way to make this something like "info" or "debug", it could be preferable.

Reply:
Txs, there is a $(info) I wasn't aware of, I'll change to that.